app/tests/src/actions.js

```diff
@@ -0,0 +1,127 @@
+import assert from 'assert'
```

Comment:
I would split the actions tests into multiple files. Actions.ts is big, and if we are going to extend it, the respective tests file for it would be even bigger and harder to navigate. So I would suggest to have an "actions" folder and multiple test files within it

For now, I would split it into "byron" and "shelley" and possibly "common" tests

Reply:
As Byron & Shelley tests will often share testdata and other preparations for testing the same function, I split it rather by functional groupings (wallet - things around login, transaction - things around transactions & fees, ...)